bolt-servlet/src/test/java/samples/OAuthSample.java

```diff
@@ -9,16 +12,36 @@
 public class OAuthSample {
 
     public static void main(String[] args) throws Exception {
-        App app = new App().asOAuthApp(true);
+        AppConfig config = ResourceLoader.loadAppConfig("appConfig_oauth.json");
+        App app = new App(config).asOAuthApp(true)
+                // Enable built-in tokens_revoked / app_uninstalled event handlers
+                .enableTokenRevocationHandlers();
```

Comment:
Any suggestions to this naming are welcomed!

Reply:
It's a tricky name, but I don't have any better suggestions. It made sense to me, so I'm okay with it!